Foundation/URLSession/NativeProtocol.swift

```diff
@@ -260,8 +260,29 @@ internal class _NativeProtocol: URLProtocol, _EasyHandleDelegate {
     }
 
     func seekInputStream(to position: UInt64) throws {
-        // We will reset the body source and seek forward.
-        NSUnimplemented()
+        // We will reset the body sourse and seek forward.
```

Comment:
Typo: sourse

Reply:
I won't confess to have fully grokked this code, but I don't see how this method can correctly implement seekInputStream if it completely ignores the position parameter. What am I missing? Or is seeking to the position not part of the contract?

Reply:
@jdberry sorry for so long delay, and thanks for your comment! I gave a possible resolution of the problem below. 👍

Reply:
We just need to add seeking forward here.